students/minha/westagram/users/views.py

```diff
@@ -1,3 +1,63 @@
 from django.shortcuts import render
 
 # Create your views here.
+import json
+import re
+
+from django.http            import JsonResponse  
+from django.views           import View 
+
+from users.models           import User
```

Comment:
가장 긴 단어 기준으로 1칸만 띄어 주세요.

```suggestion
from django.http  import JsonResponse  
from django.views import View 

from users.models import User
```